ElementX/Sources/UITests/UITestsSignalling.swift

```diff
@@ -29,6 +29,11 @@ enum UITestsSignal: Codable, Equatable {
     
     /// Posts a notification.
     case notification(name: Notification.Name)
+    
+    /// Asks for the next preview
+    case nextPreview
+    case nextPreviewReady(name: String)
+    case noMorePreviews
```

Comment:
I realised this would be a bit of a refactor, but could we do the same here as we've done with the timeline above and have

```suggestion
    case accessibilityAudit(AccessibilityAudit)
    enum AccessibilityAudit: Codable, Equatable {
        /// Ask the app for the next preview.
        case nextPreview
        /// Tell the test runner about a loaded preview.
        case nextPreviewReady(name: String)
        /// Tell the test runner that there are no more previews.
        case noMorePreviews
    }
```